src/test/java/htsjdk/samtools/util/HttpUtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package htsjdk.samtools.util;
+import java.io.IOException;
+import java.net.URL;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import htsjdk.HtsjdkTest;
+
+public class HttpUtilsTest extends HtsjdkTest {
+    @DataProvider(name = "existing_urls")
```

Comment:
If you only have a single test case, it would be easier to create a static field of type `URL` than use a data provider.